test/static/bundle-module.js

```diff
@@ -336,10 +336,13 @@ module.exports = function () {
 
         this.hostIsNested = this.hostEl.parentNode !== this.el;
 
-        // if the host el is nested one level deep we need a reference to it's container
+        // if the host el is nested we need a reference to it's container
```

Comment:
Looks like this generated file needs rebuilding.

Reply:
Does it only get built for tests? Should it even be in the repo?

Reply:
I believe it was the only way I could get Karma and Lasso to do what I wanted. This is going back about 2 years and probably needs revisiting when I have time.